### PR TITLE
Update docker start procedure  (Draft in progress)

### PR DESCRIPTION
**Pull request by keegoid-nr**
- Docker Engine must be installed first
- The Linux Terminal must be used for all platforms (including Windows)
- Clarify common permissions issues

### CODE REVIEW

**Comment by keegoid-nr** [comment]
Requesting review from @bpeckNR.

File: src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx [hunk 1]
```diff
@@ -340,31 +340,22 @@ Below are the applicable Docker or Kubernetes instructions to start the job mana
     title={<><img src={syntheticDocker} title="Docker icon" alt="Docker icon" style={{ height: '35px', width: '40px' }}/> Docker start procedure</>}
   >
     1. Locate your [private location key](#private-location-key).
-    2. Ensure you've enabled [Docker dependencies](#sandboxing-and-docker-deps) for sandboxing and [installed](#install-update) synthetics job manager on your system.
-    3. Run the appropriate script for your system. Tailor the common default `/var/run/docker.sock` in the following examples to match your system.
+    2. Ensure you've enabled [Docker dependencies](#sandboxing-and-docker-deps) for sandboxing and [installed Docker Engine](https://docs.docker.com/engine/install/) for your platform. If running as a [non-root user](#run-non-root), also complete the Docker Post-installation steps to [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
+    3. Run the following command from a Linux Terminal. Tailor the common default for `/var/run/docker.sock` to match your system. For Windows, install a Linux distribution in WSL 2 and run the command from there.
 
-       * Linux/macOS:
-
```
Comment (mderemer-nr) [inline src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx:347]: Do you want the gap between os and the docker run cmd? Linux bullet point has a gap windows doesn't. Not sure if this matters on the rendered page.

**Review by mderemer-nr — APPROVED** [review]

File: src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx (hunk 1 above)
Comment (keegoid-nr) [reply src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx:347]: I removed the bullet points completely. See changes on right.

**Comment by paperclypse** [comment]
@keegoid-nr how does this relate to PR #10866?

**Comment by austin-schaefer** [comment]
Closing out all open 2022 PRs, feel free to re-open if there's active work here or this becomes active again!

**Pull request closed without merging**